src/logger.js

```diff
@@ -1,14 +1,44 @@
+import chalk from 'chalk'
+
 const LOG_LEVELS = {
   debug: 1,
-  info: 2,
-  silent: 3
+  protractor: 2,
+  info: 3,
+  silent: 4
 }
 
-export default function (levelName, message) {
-  let currentLevel = LOG_LEVELS[process.env.PROTRACTOR_FLAKE_LOG_LEVEL] || LOG_LEVELS.info
-  let incomingLevel = LOG_LEVELS[levelName]
+const DEFAULT_COLOR = 'magenta'
 
-  if (incomingLevel >= currentLevel) {
-    process.stdout.write(message)
+class Logger {
+  constructor (color) {
+    if (color === true) {
```

Comment:
If we make sure that color is either a valid color or `false` we could just change this to

```javascript
this.color = color || DEFAULT_COLOR
```